ls1intum/Artemis
# Prevent creation of empty textblocks

### Motivation and Context
When assessing text-exercises with multiple sentences with spaces or newlines in between 2 textblocks, an empty textblock with this space/newline in between got created in the database. This PR fixes this problem

### Description
- Introduced a check to prevent the creation of empty (only spaces/newlines) textblocks

### Steps for Testing
1. Create a text-exercise
2. Create a submission with multiple sentences separated by ". " (mind the space after the ".")
3. Assess the submission using the automatic selection of text-blocks
4. Submit the assessment
5. Check the database (Table "text-block") for empty entries
